data/data-pipeline/.vscode/launch.json

```diff
@@ -9,63 +9,100 @@
       "type": "python",
       "request": "launch",
       "module": "data_pipeline.application",
-      "args": ["score-run"]
+      "args": [
+        "score-run"
+      ]
     },
     {
       "name": "Data Cleanup",
       "type": "python",
       "request": "launch",
       "module": "data_pipeline.application",
-      "args": ["data-cleanup"]
+      "args": [
+        "data-cleanup"
+      ]
     },
     {
       "name": "Census Cleanup",
       "type": "python",
       "request": "launch",
       "module": "data_pipeline.application",
-      "args": ["census-cleanup"]
+      "args": [
+        "census-cleanup"
+      ]
     },
     {
       "name": "Download Census",
       "type": "python",
       "request": "launch",
       "module": "data_pipeline.application",
-      "args": ["census-data-download"]
+      "args": [
+        "census-data-download"
+      ]
     },
     {
       "name": "Score Full Run",
       "type": "python",
       "request": "launch",
       "module": "data_pipeline.application",
-      "args": ["score-full-run"]
+      "args": [
+        "score-full-run"
+      ]
+    },
+    {
+      "name": "Score Geo",
```

Comment:
what does this do exactly? wondering if there's a verb we can use that'll make this clearer

Reply:
That is what we called the "Combine" lambda function. Should we call it "Combine Score and GeoJSON"?